rholang/src/main/scala/coop/rchain/rholang/interpreter/Interpreter.scala

```diff
@@ -103,7 +103,7 @@ object Interpreter {
 
   def evaluate(runtime: Runtime, normalizedTerm: Par): Task[EvaluateResult] = {
     implicit val rand      = Blake2b512Random(128)
-    val evaluatePhlosLimit = Cost(Integer.MAX_VALUE)
+    val evaluatePhlosLimit = Cost(Integer.MAX_VALUE) //This is OK because evaluate is not called on deploy
```

Comment:
That is quite an assumption. Methods should not assume where they're called from, as this changes quickly, easily, and leads to horrible bugs. In this case its not that bad though, as the signature lacking the phlo is immediately suspicious. Which is, to me, what makes it OK :)